### PR TITLE
docs: add the resource inventory

### PR DESCRIPTION
**Pull request by zimbatm**

### CODE REVIEW

File: docs/inventory.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+(for ofborg)
+
+## IRC logging bot
+
+owner: SamuelDR
+https://logs.nix.samueldr.com/nixos/
+
+nick:
+
+    {`-`}_
```
Comment (samueldr) [inline docs/inventory.md]: The registered nickname is <code>{\`-\`}</code>, it sometimes restarts after network events (splits, ping timeouts) and will not recover its registered nickname (as there is no code to do so).

As its nickname may cause issues with markdown implementations, you can use this to inline it:

    <code>{\`-\`}</code>

Furthermore, while not publicly advertised, but publicly shared in the past, this is the overlay used for the IRC bot.

 * https://gitlab.com/samueldr.nix/overlays/irclogger

File: docs/inventory.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+owner: Graham
+
+## Hetzner Cloud
+
+owner: Graham
+(for ofborg)
+
+## IRC logging bot
+
+owner: SamuelDR
```
Comment (samueldr) [inline docs/inventory.md]: You can use samueldr, or my full name Samuel Dionne-Riel. (I try to keep away from capitalizing the nickname as the "dr" part may cause confusions with the "doctor" title.)

File: docs/inventory.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+## YouTube channel for recorded talks
+
+owner: Cillian?
+url: https://www.youtube.com/channel/UCjqkNrQ8F3OhKSCfCgagWLg
+
+
+## NixOS community wiki + bot
+
+owner: samueldr
```
Comment (samueldr) [inline docs/inventory.md]: While I have been marked as *maintainer* for [nix-community/wiki](https://github.com/nix-community/wiki), the wiki and the IRC bot is maintained by @fadenb. The [NixOS_Wiki:About](https://nixos.wiki/wiki/NixOS_Wiki:About) may be a relevant link for this section.

File: docs/inventory.md (hunk 1 above)
Comment (zimbatm) [reply docs/inventory.md]: fixed

File: docs/inventory.md (hunk 3 above)
Comment (zimbatm) [reply docs/inventory.md]: thanks again! fixed

**Comment by vcunat** [comment]
You can add my bunch of crappy HW :-)  All has DNS `*.cunat.cz`, only IPv6 from outside (ssh + ping), located at my home:
 - `t4a`, `t4b`: 4-core i5, working as regular Hydra slaves for a few weeks now;
 - `t2m`: mac mini (2-core core2duo), administration handed over to @grahamc, about to be set up as slave;
 - `t2a`: 2-core ("core"-microarch Pentium) for the `metrics` job, set up already, waiting to be plugged as Hydra slave.

**Pull request merged**